fossology/fossology
# feat(project): add page of project create, delete, edit and browse. add a new option of project in upload page

## Description

Add page of project create, delete, edit and browse. add a new option of project in upload page

### Changes

There are four new pages: project_create, project_delete, project_edit and BrosweProject. There is also a new option in upload_file page to determine which project the upload should be in.

## How to test

Just click the corresponding menu-items in the main menu and you could see these pages. Most of them are similar to folder pages and you could check their function.

<a href="https://gitpod.io/#https://github.com/fossology/fossology/pull/2313"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
